### ISSUE TITLE
How can i remove an instance from my phone app ?

### ISSUE THREAD

**Issue by UnclePhil** [Looking4Cache/node-red-contrib-remote#6]
Add is very convenient, but when you make some development for friends or customers, you need also to have the option of removing an instance from your (test) phone.

Is there any options to do this removal ??

Best regards
Ph Koenig

**Comment by Looking4Cache** [comment]
Hi,

just swipe on the instance from right to left to delete it. Maybe I need a info on Android, as this is very common on iOS, but not on Android.

Best regards,
Thorsten

**Comment by UnclePhil** [comment]
So simple...
Yes, you need to add one line in help of the application.

Thank's

**Comment by billjuv** [comment]
I agree that a mention of “swipe left to delete” in help would be useful. Took me a half hour to stumble across this.  Another “duh” moment.

otherwise, a great app!

**Comment by Looking4Cache** [comment]
In version 2.0.0 the deletion on Android is done with a long press. Also it is now added to the help file.
